src/pages/Home/Home.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useSelector, useDispatch } from 'react-redux';
+import Countries from '../../components/Countries/Countries';
+import SearchInput from '../../components/SearchInput/SearchInput';
+import HeroPage from '../../components/HeroPage/HeroPage';
+import { getCountryCases } from '../../redux/covid/Covid';
+
+const Home = () => {
+  const [search, setSearch] = useState('');
+  const allCountries = useSelector((state) => state.covid);
+  const dispatch = useDispatch();
+  const id = uuidv4();
+
+  const fetchCoutries = async () => {
+    const res = await fetch('https://disease.sh/v3/covid-19/countries');
+    const data = await res.json();
+    const countries = data.map((country) => ({ ...country, id }));
+    dispatch(getCountryCases(countries));
+  };
```

Comment:
@medaminedev66 Thank you for your feedback. However, if would have been nice if you had taken a good look at the redux directory because if you did, you would have noticed that 'getCounteryCases is an action used to dispatch the data from the API to the redux store. 

There are other ways to aside the example you pointed out which is also a getter practice. 
![image2](https://user-images.githubusercontent.com/48631109/160230805-c116ce75-92d8-46d1-8161-4448f35acb9a.PNG)